pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
> I'm definitely not crazy about the readability here,

Definitely not. I'm intentionally being basic with our API abstractions. It's on my mind, though. See #40 as an example of some of the thought I've put into this. (The comments on that issue are already a bit obsolete. I've been wondering if we can do anything nice with named tuples to cut down on those function signatures.)